src/app/modules/example-feature/example.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface ExampleInterface {
```

Comment:
I think this should be moved inside the `state` folder

Reply:
Done